tgui/packages/tgui/styles/interfaces/Playtime.scss

```
@@ -0,0 +1,35 @@
+@use '../base.scss';
+
+.PlaytimeInterface {
+  td {
+    padding: 0;
+    height: 34px;
+  }
+  table {
+    border: 1px solid white;
+    border-radius: 4px;
+  }
+
+  tr {
+    border-bottom: 1px solid white;
```

```suggestion
  .Table__cell {
    border-bottom: 1px solid white !important;
  }
```
to override the theme